xrpl4j-integration-tests/src/test/java/org/xrpl/xrpl4j/tests/AccountTransactionsIT.java

```diff
@@ -74,35 +76,37 @@ public void listTransactionsPagination() throws JsonRpcClientErrorException {
     // known ledger index range for this account that is known to have exactly 748 transactions
     LedgerIndexBound minLedger = LedgerIndexBound.of(61400000);
     LedgerIndexBound maxLedger = LedgerIndexBound.of(61487000);
+    Set<Hash256> transactionHashes = new HashSet<>();
+
     AccountTransactionsResult results = mainnetClient.accountTransactions(
       AccountTransactionsRequestParams.builder(minLedger, maxLedger)
         .account(MAINNET_ADDRESS)
         .build()
     );
     assertThat(results.transactions()).isNotEmpty();
     assertThat(results.marker()).isNotEmpty();
+    results.transactions().forEach(transaction -> transactionHashes.add(transaction.resultTransaction().hash()));
```

Comment:
Per discussion with @nkramer44, we'll revert this back to the original and per the comments below -- this test will run against a variety of additional environments.